core/src/main/java/io/neow3j/transaction/TransactionAttribute.java

```diff
@@ -71,18 +71,6 @@ public static TransactionAttribute deserializeAttribute(BinaryReader reader)
         }
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (!(o instanceof TransactionAttribute)) {
-            return false;
-        }
-        TransactionAttribute that = (TransactionAttribute) o;
-        return getType() == that.getType();
-    }
-
```

Comment:
Yes, you're right. We should add equals method to the concrete classes.

I removed this equals method, because it falls short when comparing concrete TransactionAttribute types that have other values besides their type. Comparing them would've fallen back to this equals method of the abstract class, resulting in only comparing the types of the attributes (i.e., any OracleResponseAttribute would be equal to any other OracleResponseAttribute).

Reply:
Since neow3j is not really used for oracle responses, I let it fall back to the `Object.equals` method when comparing it with `equals` (which will do a comparison by reference) and take the easy path of not write and test a separate `equals` method for it. This makes me wonder if we should even provide the class `OracleResponseAttribute` ...

Reply:
Regardless of what we'll do with the `OracleResponseAttribute`, I'll add an `equals` method to the `HighPriorityAttribute`.

Reply:
Makes sense to me 👌